GameOfLife/GameOfLife/Services/FieldOperations.cs

```diff
@@ -0,0 +1,221 @@
+                    {
+                        gameField.GameField[_coordinateX, _coordinateY] = AliveCellSymbol;
+                    }
+                    else
+                    {
+                        gameField.GameField[_coordinateX, _coordinateY] = DeadCellSymbol;
+                    }
+                }
+                else
+                {
+                    Stop = false;
+                    break;
+                }
+            }
+            return gameField;
+        }
+
+        /// <summary>
+        /// Cell amount and coordinates are generated automatically and randomly.
+        /// </summary>
+        /// <param name="gameField">An instance of the GameFieldModel class.</param>
+        /// <returns>Returns an instance of the GameFieldModel class with alive cells randomly seeded in its field.</returns>
+        public GameFieldModel RandomSeeding(GameFieldModel gameField)
+        {
+            Random random = new();
+            int aliveCellCount = random.Next(1, gameField.Length * gameField.Width);
+            int randomX, randomY;
+
+            for (int i = 1; i <= aliveCellCount; i++)
+            {
+                randomX = random.Next(0, gameField.Length);
+                randomY = random.Next(0, gameField.Width);
+
+                if (gameField.GameField[randomX, randomY] != AliveCellSymbol)
+                {
+                    gameField.GameField[randomX, randomY] = AliveCellSymbol;
+                }
+                else
+                {
+                    random = new();
+                }
+            }
+            return gameField;
+        }
+
+        /// <summary>
+        /// Method to choose a cell pattern from the premade library.
+        /// </summary
+        /// <param name="gameField">An instance of the GameFieldModel class.</param>
+        /// <param name="gliderGunMode">Parameter to show whether the glider gun mode is on.</param>
+        /// <param name="gliderGunType">Parameter that represents the chosen type of the glider gun.</param>
+        /// <returns>Returns an instance of the GameFieldModel class with a library object seeded in its field.</returns>
+        public GameFieldModel LibrarySeeding(GameFieldModel gameField, bool gliderGunMode, int gliderGunType)
+        {
+            ConsoleKey libraryChoice;
+
+            while (true)
+            {
+                if (gliderGunMode)
+                {
+                    switch (gliderGunType)
+                    {
+                        case 1:
+                            gameField = _library.SpawnGosperGliderGun(gameField, 1, 1);
+                            break;
+
+                        case 2:
+                            _library.SpawnSimkinGliderGun(gameField, 0, 16);
+                            break;
+                    }
+                    Console.Clear();
+                    return gameField;
+                }
+                if (_inputProcessor.WrongInput)
```

Comment:
Here and in other places, add empty line after closing block.

Reply:
I am not sure what the  'closing block' is, where exactly should I put empty lines? Between the 'if' blocks?

Reply:
In C#, a code block is a group of lines of code between curly braces {}.
After closing brace there should be empty line except block is followed by closing brace of enclosing block.

So:
![image](https://user-images.githubusercontent.com/103197809/166472189-62118378-ca40-4cf2-bd86-2322c381c587.png)
On image red lines where no empty line, green line where empty line should be.

Reply:
So there should be no empty lines between 'if' and 'else? And also, should I add an empty line in such a situation?
<img width="259" alt="image" src="https://user-images.githubusercontent.com/98710955/166871527-01e2ed20-669d-4b27-9b2d-1e26aeb12033.png">

Reply:
Added necessary empty lines.